src/schema/v1/fields/__tests__/money.test.ts

```diff
@@ -3,12 +3,15 @@ import { amount } from "../money"
 describe(amount, () => {
   let amountCents: number
   let args: any
+  let obj: any
+
   beforeEach(() => {
+    obj = {}
     args = { symbol: "$" }
     amountCents = 1234
   })
 
-  const getResult = () => amount(() => amountCents).resolve({}, args)
+  const getResult = () => amount(() => amountCents).resolve(obj, args)
```

Comment:
Nit-pick: Instead of defining `obj` and `args` as `let` variables, it seems like it would be simpler to just pass parameters to `getResult`?

Reply:
Same goes for `amountCents`.

Reply:
@sweir27 Nice, thanks 🙏